example/tests/test_backends.py

```diff
@@ -0,0 +1,56 @@
+import json
+
+from rest_framework.test import APITestCase
+
+from ..models import Blog, Entry
+
+ENTRIES = "/nopage-entries"
```

Comment:
we shouldn't build routes manually but instead use [django reverse resolver](https://docs.djangoproject.com/en/2.1/ref/urlresolvers/#reverse)

In this case this would be `reverse('nopage-entry')`. It will be then a full path of the testserver url.

This needs to be done in setUp or what I usually do in each single test as it is more implicit.

Reply:
```
django.urls.exceptions.NoReverseMatch: Reverse for 'nopage-entry' not found. 'nopage-entry' is not a valid view function or pattern name.
```

Reply:
maybe `nopage-entry-list`